core/primitives/src/bandwidth_scheduler.rs

```diff
@@ -73,8 +73,63 @@ pub struct BandwidthRequestsV1 {
     ProtocolSchema,
 )]
 pub struct BandwidthRequest {
+    /// Requesting bandwidth to this shard.
     pub to_shard: u8,
-    // TODO(bandwidth_scheduler) - store requested bandwidth values inside the BandwidthRequest
+    /// Bitmap which describes what values of bandwidth are requested.
+    pub requested_values_bitmap: BandwidthRequestBitmap,
+}
+
+/// There are this many predefined values of bandwidth that can be requested in a BandwidthRequest.
+pub const BANDWIDTH_REQUEST_VALUES_NUM: usize = 40;
+
+/// Bitmap which describes which values from the predefined list are being requested.
+/// The nth bit is set to 1 when the nth value from the list is being requested.
+#[derive(
+    BorshSerialize,
+    BorshDeserialize,
+    serde::Serialize,
+    serde::Deserialize,
+    Debug,
+    Clone,
+    PartialEq,
+    Eq,
+    ProtocolSchema,
+)]
+pub struct BandwidthRequestBitmap {
+    pub data: [u8; BANDWIDTH_REQUEST_BITMAP_SIZE],
```

Comment:
Shouldn't that be divided by 8? 
`pub data: [u8; BANDWIDTH_REQUEST_BITMAP_SIZE / 8 ]`

Reply:
ignore it, I only saw the NUM const and I thought it's this one